src-tauri/src/northstar/install.rs

```diff
@@ -96,6 +96,22 @@ async fn do_install(
     log::info!("Extracting Northstar...");
     extract(nfile, std::path::Path::new(&extract_directory))?;
 
+    if game_install.profile != "R2Northstar" {
+        // We are using a non standard Profile, we must:
+        // - move the DLL
+        // - rename the Profile
+
+        // Move DLL into default Profile
+        let old_dll_path = format!("{}/Northstar.dll", extract_directory);
+        let new_dll_path = format!("{}/R2Northstar/Northstar.dll", extract_directory);
+        std::fs::rename(old_dll_path, new_dll_path)?;
+
+        // rename default Profile
+        let old_profile_path = format!("{}/R2Northstar/", extract_directory);
+        let new_profile_path = format!("{}/{}/", extract_directory, game_install.profile);
+        std::fs::rename(old_profile_path, new_profile_path)?;
```

Comment:
Looking just at the diff, I was a bit confused why you are moving the DLL into the R2Northstar profile and then move rename the entire profile folder until I realised that this was still during initial extract process ^^"

Not sure if a comment or sth would help help this make it clearer...